docs/topics/multiplatform/multiplatform-compatibility-guide.md

```diff
@@ -20,17 +20,18 @@ When configuring your project, check the compatibility of a particular version o
 
 | Kotlin Multiplatform plugin version | Gradle                                 | Android Gradle plugin           | Xcode   |
 |-------------------------------------|----------------------------------------|---------------------------------|---------|
-| 2.1.0                               | %minGradleVersion%–%maxGradleVersion%* | 7.4.2–%maxAndroidGradleVersion% | %xcode% |
+| 2.1.20                              | %minGradleVersion%–%maxGradleVersion%* | 7.4.2–%maxAndroidGradleVersion% | %xcode% |
```

Comment:
Should * be at the end? withJava() is deprecated in this release and user could have KMP project without deprecations.